test/framework/src/main/java/org/elasticsearch/test/rest/yaml/restspec/ClientYamlSuiteRestSpec.java

```diff
@@ -80,9 +80,9 @@ public boolean isClientParameter(String name) {
      * Parses the complete set of REST spec available under the provided directories
      */
     public static ClientYamlSuiteRestSpec load(String classpathPrefix) throws Exception {
-        Path dir = PathUtils.get(ClientYamlSuiteRestSpec.class.getResource(classpathPrefix).toURI());
         ClientYamlSuiteRestSpec restSpec = new ClientYamlSuiteRestSpec();
         ClientYamlSuiteRestApiParser restApiParser = new ClientYamlSuiteRestApiParser();
+        Path dir = PathUtils.get(ClientYamlSuiteRestSpec.class.getResource(classpathPrefix).toURI());
```

Comment:
any reason these got moved around here? Seems like the imports and the `dir` are exactly the same, albeit just moved a few lines.

Reply:
I believe this is because he tried to make a change to this file but it didn't work out and he reversed it by hand. @javanna , can you entirely revert the changes to this file?